### PR TITLE
docs: updated README based on my getting-started experience.

### PR DESCRIPTION
**Pull request by rjhuijsman**
Updated README with information that would have helped me get started.
Updated Gradle version because Android Studio was adamant that that was a good idea.
Added myself to the credits, as per contributor guidelines.

## Screenshots
None

## How did you test the change?

- [ ] iOS Simulator
- [ ] iOS Device
- [X] Android Simulator
- [ ] Android Device
- [ ] `curl` to a dev App Engine server
- [ ] other, please describe

## Checklist:

- [X] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md) and verified that all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE).

### CODE REVIEW

**Review by theswerd — CHANGES_REQUESTED** [review]
Please remove changes to `client/android/build.gradle` and `client/android/gradle/wrapper/gradle-wrapper.properties`

File: client/android/build.gradle [hunk 1]
```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:4.1.2'
```
Comment (theswerd) [inline client/android/build.gradle:9]:
```suggestion
        classpath 'com.android.tools.build:gradle:3.5.0'
```

File: client/android/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Fri Jun 23 08:50:38 CEST 2017
+#Tue Mar 09 22:46:39 PST 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (theswerd) [inline client/android/gradle/wrapper/gradle-wrapper.properties:6]:
```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
```

File: client/android/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
-#Fri Jun 23 08:50:38 CEST 2017
+#Tue Mar 09 22:46:39 PST 2021
```
Comment (theswerd) [inline client/android/gradle/wrapper/gradle-wrapper.properties:1]:
```suggestion
#Fri Jun 23 08:50:38 CEST 2017
```

**Comment by rjhuijsman** [comment]
> Please remove changes to `client/android/build.gradle` and `client/android/gradle/wrapper/gradle-wrapper.properties`

Done! Mind teaching a man to fish by explaining why you think it's better left as-is?

File: client/README.md [hunk 4]
```diff
@@ -2,13 +2,32 @@
...
+
+- Run `File -> Sync Project with Gradle Files` to re-sync; this sync will succeed.
+- Start your Android Virtual Device (AVD): `Tools` -> `AVD Manager` -> (on listed device) `Actions` -> `Run`.
+
+You can now run the app by running:
 
 ```
 cd client
-flutter run
+flutter run --flavor prod
```
Comment (theswerd) [inline client/README.md:30]: For development purposes, users should use the staging version in order to allow us access to the server's debug logs.
```suggestion
flutter run --flavor stag
```

**Review by theswerd — CHANGES_REQUESTED** [review]

**Review by theswerd — APPROVED** [review]
LGTM

**Comment by rjhuijsman** [comment]
Thanks @theswerd !

I don't have the button to actually do the merge - who should I poke?

**Comment by theswerd** [comment]
@rjhuijsman Sorry about that - I can merge

**Pull request merged**